stdlib/discreteDistributions.hk

```diff
@@ -0,0 +1,58 @@
+# Hakaru Standard Library
+# Discrete Distributions: 
+#	definitions for distributions with discrete outcomes
+# author: Mahmoud Khattab
+
+####################
+# Discrete Uniform #
+####################
+
+# Discrete rectangular distribution.
+# Distribution ranges from 0 to n (inclusive) all with the same probability
+def rectangular(n nat):
```

Comment:
This needs a better name.

Reply:
The name is based on [this document](http://www.math.wm.edu/~leemis/chart/UDR/PDFs/Rectangular.pdf) from [this website](http://www.math.wm.edu/~leemis/chart/UDR/UDR.html) which Dr. Carette supplied my capstone group with to base the standard library on.

I don't think it's a good name either but I'm not sure what else to call it. I've tried to looking up an alternate name with no luck

Reply:
Maybe uniformRange?

Reply:
How about `natUniform`, and rename `discreteUniform` to `intUniform`?

Reply:
Or we could change uniform to take 2 arguments like Lebesgue?

Reply:
:+1: for natUniform and intUniform. In a perfect world, we would have polymorphism and uniform would have a type like Semiring a => a -> a -> Measure a